pkg/gui/view_helpers.go

```diff
@@ -288,6 +290,11 @@ func (gui *Gui) getCommitMessageView(g *gocui.Gui) *gocui.View {
 	return v
 }
 
+func (gui *Gui) getPushPassUnameView(g *gocui.Gui) *gocui.View {
```

Comment:
Given that these helper methods expect the panels to always exist (i.e. they are first class panels that never get deleted), and given that they swallow errors, I don't think such a helper method should exist for the push-pass-uname panel, unless we end up changing the code so that it only ever gets moved to the back rather than deleted.